### PR TITLE
Misc loot crate loot balancing.

### PR DESCRIPTION
**Pull request by ThatFiveGuys**
## About the PR 
This pr partially changes the rewards for several crates. For t3 science crates, instead of just a telewand, it will now roll between a telewand, artifact activator, artgun, meleeart, and a handheld forcefield art. This should reduce the chance of a telewand from crates greatly. This pr swaps the atomic cell and the disruptor cell with the standard pod wars cell (300 pu 15 self recharge) and a 400 pu cell. Disruptor cell and atomic cell are pretty useless for most weapons, it doesnt make sense for these to be from a t3 crate. The telegun has been removed from tier 2 military crates, and the predator rifle from tier 2 military crates now come with a fission cell (200 pu 20 self recharge), and has half the fire cost. The one million credits from crates have been swapped with three stacks of one hundred thousand credits.

## Why's this needed? 

**Voltron is pretty bad in regard of crates, its just Id need to add a lot more content to justify removing them from crates right now. So uhh yeah thats why this pr doesnt do that before that gets brought up.**

Most of this pr is to simply nerf the usefulness of loot crates. A 7 traitor credit weapon being crew accessible is pretty bad, hence telegun removal. Laser rifle was a mainstay of predator, biggest strength is its very high self recharge. In the context of loot crates this makes it basically a better phaser with less shots but with self recharge and more expensive. Telewands are very powerful, them being less common is good. The atomic cells and disruptor cells were basically useless, especially considering this is supposed to be a tier 3 drop. Economy is shifting away slowly from low effort get rich quick shit to now its a lot harder to get rich, the three hundred credits instead of one million should accompany itself nicely with this trend.

## Changelog

```changelog
(u)Ikea
(*)Loot crates contents have undergone some balance changes, check minor changes for more details.
(*)Predator rifles now come with half the charge and half the self recharge rate, but with half the firing costs.
(+) Crates will now pull from a pool of handheld arts instead of just telewands. Telegun has been completely removed from loot crates. The atomic cell and disruptor cell have been replaced with better cells. The one million crates from loot crates has been replaced with three hundred thousand credits.
```

### CODE REVIEW

**Comment by bugreportercpx** [comment]
I don't like the economy changes.

Unmarked Tier 3 crates are exceedingly rare, and getting the million credits from them is even rarer. If you take this away, as well as the proposed changes to the wish granter artifact, certain content (such as plutonium cores, medusa system, etc) goes from being extremely rare to completely inaccessible.

I think it would make make more sense to make economic changes in a comprehensive overhaul that looks at what's accessible at traders rather than slowly start to effectively remove content.

**Comment by ThatFiveGuys** [comment]
Its important to note that like, i dont think being able to rng get very expensive shit 5 minutes in is good. Its really not like this expensive shit depends on the rng get rich quick methods too. Stuff like starstones still exist. The idea is that the more expensive something is, the more time it takes to get. If you want to go for those things, you should be going for stuff like starstones. Low effort ways to amass fortunes very quickly is bad.

**Comment by Momoberry** [comment]
I think the rare once in 500 round occurrences of 10 minute 40mm pod shells are honestly fine. I like the changes outside of the economy ones. Right now money is basically useless for crew, and making money without being QM is essentially never going to happen outside of crates. Even with QM, making money is tedious and there's nothing to actually buy with it. 

The most expensive things require in hand cash, not in a budget, and require you to be an antagonist to access the areas without cheesing it. It's hard enough as it is, and removing these rare happenings makes it pseudo impossible unless you want to make people work for 50 minutes just to even make money the actual money.... and that's with full access to the qm and budget, with nobody interfering with you. And then you have to have access to the budget and transferring to even move that budget, which is only on two jobs naturally. Maybe you can do it with packets or comp? I have no inkling how though.

**Comment by ThatFiveGuys** [comment]
Important to note that the only real things that are over 300k is like, medusa and plutonium core. Medusa is very strong, and Im fine with it not being something thats easy to get, considering you can fire with medusa equipped, especially on ocean maps its an incredibly strong item. Plutonium core does something I cannot disclose here, and yeah I am 100% fine with that being very rare. This change just prevents you from being able to get basically every good item incredibly quickly into the round (because lets be honest most crates arent gotten 30 minutes in, theyre teleported into telesci 5 minutes in)

**Comment by ThatFiveGuys** [comment]
Just so this is noted here, debris has 31 crates and around 1/8 crates are red. This means theres around 4 red crates in debris per round, and you have around a 25% chance of getting any t3 item per round. Like for example you have a 25% chance of getting a million credits every single round just by hitting crates.

**Comment by Supermichael777** [comment]
Just remember gold bars are 35k each and  yo almost always get at least 6-7 plus cash crates plus whatever is in faint signal safe.  That almost always comes out to at least another 220k of easy money.  And that's if you can't manage to get half a hot cargo shipment or starstone

**Comment by ThatFiveGuys** [comment]
Ah I am dumb, done

File: code/modules/projectiles/laser.dm [hunk 1]
```diff
@@ -281,6 +281,9 @@ toxic - poisons
 	color_green = 0.5
 	color_blue = 0.7
 
+/datum/projectile/laser/pred/cheap // For loot crate pred rifle
+	cost = 25
+
```
Comment (Tarmunora) [inline code/modules/projectiles/laser.dm:286]: move cost adjust to parent

**Review by Tarmunora — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/laser.dm (hunk 1 above)
Comment (ThatFiveGuys) [reply code/modules/projectiles/laser.dm]: Oops forgot to commit my removal of that my bad, done.

**Review by Tarmunora — APPROVED** [review]

**Pull request merged**